File: packages/cortex/example/android/app/src/main/java/com/emotiv/cortex_example/MyApplication.java
package com.emotiv.cortex_example;

import android.app.Application;
import android.util.Log;

import com.emotiv.CortexLib;
import com.emotiv.EmotivLibraryLoader;

public class MyApplication extends Application {
    @Override
    public void onCreate() {
        Log.e("app", "run my application");
        EmotivLibraryLoader loader = new EmotivLibraryLoader(this);
        loader.load();
        super.onCreate();
    }

    @Override
    public void onTerminate() {
        super.onTerminate();
        CortexLib.stop();
    }
}

File: packages/cortex/example/android/app/src/main/kotlin/com/emotiv/cortex_example/MyApplication.kt
package com.emotiv.cortex_example

import android.app.Application
import android.util.Log
import com.emotiv.CortexLib
import com.emotiv.EmotivLibraryLoader

class MyApplication : Application() {
    override fun onCreate() {
        Log.e("app", "run my application")
        val loader = EmotivLibraryLoader(this)
        loader.load()
        super.onCreate()
    }

    override fun onTerminate() {
        super.onTerminate()
        CortexLib.stop()
    }
}
